client/src/components/LoginForm/LoginForm.js

```diff
@@ -0,0 +1,88 @@
+import { useState } from 'react'
+import axios from 'axios'
+import styled from 'styled-components/macro'
+import ButtonGreen from '../Buttons/ButtonGreen/ButtonGreen'
+import ErrorCard from '../Messages/ErrorCard/ErrorCard'
+import SuccessCard from '../Messages/SuccessCard/SuccessCard'
+
+const LoginForm = () => {
+  const [username, setUsername] = useState('')
+  const [password, setPassword] = useState('')
+  const [errorMessage, setErrorMessage] = useState('')
+  const [successMessage, setSuccessMessage] = useState('')
+
+  const handleChangeUsername = event => {
+    setUsername(event.target.value)
+  }
+
+  const handleChangePassword = event => {
+    setPassword(event.target.value)
+  }
+
+  const handleSubmit = event => {
+    event.preventDefault()
+    axios
+      .post(`/api/auth`, { username, password })
+      .then(res => {
+        setSuccessMessage('User logged in!')
+        if (errorMessage !== '') setErrorMessage('')
```

Comment:
Always use curly braces for conditional branching

Reply:
Changed!